AmplifyPlugins/Auth/AWSCognitoAuthPluginTests/AuthenticationProviderTests/AuthenticationProviderResetPasswordTests.swift

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2018-2020 Amazon.com,
+// Copyright 2018-2021 Amazon.com,
```

Comment:
Did you change this manually? I think we handle this using some script. I would prefer changing the year in all source file together as a separate PR.

Reply:
Ya, it is changed automatically together with many other files, I don't know if these changes are proper in this PR so that I reverted those should not happen in this PR, leaving only the changes in `AuthenticationProviderResetPasswordTests.swift` and `MockAWSMobileClient`